subgraph/OKX_X1_testnet-subgraph.yaml

```diff
@@ -0,0 +1,77 @@
+specVersion: 0.0.6
+description: A subgraph of Ledgity Yield data
+repository: https://github.com/LedgityLabs/LedgityYield
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: LTokenSignaler
+    network: arbitrum-one
+    source:
+      address: "0x1e1B0bB6C182eFD7d64cdAbbF3A3E315dE26E259"
+      abi: LTokenSignaler
+      startBlock: 10172276
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.7
+      language: wasm/assemblyscript
+      file: ./mapping.ts
+      entities:
+        - LToken
+      abis:
+        - name: LTokenSignaler
+          file: ../contracts/abis/LTokenSignaler.json
+        - name: LToken
+          file: ../contracts/abis/LToken.json
+      eventHandlers:
+        - event: LTokenSignalEvent(indexed address)
+          handler: handleSignaledLToken
+  - kind: ethereum/contract
```

Comment:
you can get rid of all the Pre-Mining stuff, that won't be deployed to OKX testnet